### DISCUSSION TITLE
Basler camera problems

### DISCUSSION THREAD

**Discussion by ManalKR** [bonsai-rx/bonsai#1275] [Q&A]
Hi, 

I am using the FP3002 system. For the behavioral recording we want to use Basler camera and timestamp it with the fp signal. I downloaded the pylon_Runtime 7.1.0 and at the beginning everything worked well. But now when I run the PylonCapture, Bonsai logs out immediately after hitting Start. Could you please help me with figuring out the problem?

Attached the workflow.

![image](https://user-images.githubusercontent.com/127253474/223848982-a4d0603a-e4bd-40af-bf77-56609e617129.png)

**Comment by bruno-f-cruz** [comment]
Can you include the workflow you are using, your bonsai.config file as well? Also, is there any error being printed to the console?

Cheers,
Bruno

**Reply by ManalKR** [reply]
There is no error printed, once I hit run it just closes Bonsai window.
How do I extract config file? I only have a config file of FP3002.

Here is the config file that I have, I'm not sure if this one is what you wanted:

This XML file does not appear to have any style information associated with it. The document tree is shown below.

2064-0061
ThisDevice
Both
7
60

L560
L410

12482
0
17792
32760
10
50
10
Software
EventRising
EventRising

242.5 615.5 79 75 0
![image (2)](https://user-images.githubusercontent.com/127253474/225091950-a94014a5-5270-4af7-beeb-2639ee3f9d14.png)

**Reply by bruno-f-cruz** [reply]
If you disable the stream of either camera (control +D), does the workflow crash? If so, which one needs to be enable for it to crash?

**Reply by ManalKR** [reply]
I disabled all node except for pylonCapture node and it still crashed.

**Reply by bruno-f-cruz** [reply]
Just to make sure, this includes disabling the fiber photometry node, correct?

**Reply by ManalKR** [reply]
For the parameterFile in went to pylon Viewer Camera > save features, and saved the pfs file. And it is the file I selected for the parameterFile. 
And I made sure it is the right serial number.

**Reply by ManalKR** [reply]
Yes, I disabled the FP3002 node

**Reply by bruno-f-cruz** [reply]
Ok at this point we need more information. Can you try to capture the error by trying to dump the process output to a txt file? Instructions can be found here https://bonsai-rx.org/docs/articles/cli.html#starting-a-workflow-in-application-mode under the blue text box.

**Reply by ManalKR** [reply]
Do I need Python to be installed to do this?

**Reply by bruno-f-cruz** [reply]
No. But if you do, you can also try to follow this:

https://github.com/bonsai-rx/bonsai/discussions/1162#discussioncomment-4273803

**Reply by ckosse** [reply]
Hi,

did you ever figure out what went wrong and how to fix it? I seem to run into the same problem.

Thanks,
Christin

**Reply by bruno-f-cruz** [reply]
Can you attach some information regarding the error you are getting? 
Can you also attach your `bonsai.config` file (located in the same folder as your bonsai executable)? Thanks

**Reply by ckosse** [reply]
Thanks Bruno,

maybe I am missing something very obvious but I don't see a .config file and there is no error, bonsai just closes after pressing start

I made sure I have the correct serial number, I have Pylon runtime 7.1.0 installed, I installed Open CV (just pip install with python)- I am new to this, was I supposed to somehow link Bonsai to OpenCV? 

Thanks so much for your help!

**Reply by ckosse** [reply]
... I am still not sure what was going wrong for me but for some reason switching to the general CameraCapture instead of the PylonCapture works! Thanks so much for all the helpful tips and tricks in this community :)
